### PR TITLE
[Feature] Set up db seeding

### PR DESCRIPTION
**Pull request by j-bunk**
## Notion ticket link

[Seed database with sample data](https://www.notion.so/uwblueprintexecs/Seed-database-with-sample-data-5619a2ae9c8f4f3e8a718b52beff5125)

## Implementation description
* The employees table in your dev db can now be populated with sample data by running the command `npx prisma db seed --preview-feature`
* Separate function to seed dev db and prod db
* Followed Prisma's [seed database guide](https://www.prisma.io/docs/guides/database/seed-database)

## Notes
* See my comments below for things I thought I'd clarify (https://github.com/uwblueprint/richmond-centre-for-disability/pull/13#discussion_r654075064, https://github.com/uwblueprint/richmond-centre-for-disability/pull/13#discussion_r654075774)

## Checklist
- [x] My PR name is descriptive, is in imperative tense, and starts with one of the following: `[Feature]`,`[Improvement]` or `[Fix]`,
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the RCD team on GitHub, or specific people who are associated with this ticket

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
     "lint:eslint": "eslint '**/*.{js,jsx,ts,tsx}' --cache --format stylish",
     "fix": "yarn fix:eslint && yarn fix:prettier",
     "fix:prettier": "prettier --write '**/*.{js,jsx,ts,tsx}'",
-    "fix:eslint": "eslint '**/*.{js,jsx,ts,tsx}' --format stylish --fix"
+    "fix:eslint": "eslint '**/*.{js,jsx,ts,tsx}' --format stylish --fix",
+    "ts-node": "ts-node --compiler-options \"{\\\"module\\\":\\\"commonjs\\\"}\""
```
Comment (j-bunk) [inline package.json:16]: The compiler options were added to make the `prisma db seed` command work with Next.js as mentioned in the [prisma docs](https://www.prisma.io/docs/guides/database/seed-database#compiler-options)

File: prisma/seed.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
+import prisma from '../prisma/index'; // Relative path required, path aliases throw error with seed command
```
Comment (j-bunk) [inline prisma/seed.ts:1]: Path aliases don't seem to work with the seed command so I had to use relative paths. I think it has to do with the fact that the seed command is run using `ts-node` and we'd need to use something like [tsconfig-paths](tsconfig-paths) to get path aliases to work with it

File: prisma/seed.ts [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+const main = async () => {
+  if (process.env.NODE_ENV === 'production') {
+    await prodSeed();
+  } else {
+    await devSeed();
+  }
+};
+
+const prodSeed = async () => {
+  return;
```
Comment (j-bunk) [inline prisma/seed.ts:13]: @OustanDing Wasn't sure what to populate the prod db with so the `prodSeed`  function doesn't do anything atm. I noticed in the (dancefest seed file)[https://github.com/uwblueprint/dancefest-web/blob/staging/prisma/seed.js] they seeded their prod admin table with one member from the team, lmk if you want me to do the same!

File: prisma/seed.ts (hunk 2 above)
Comment (OustanDing) [reply prisma/seed.ts:1]: Good note

File: prisma/seed.ts (hunk 3 above)
Comment (OustanDing) [reply prisma/seed.ts:13]: Could you please just add me as the sole prod user? Thanks!

File: prisma/seed.ts [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+  const employeeUpserts = [];
+
+  for (const employee of employees) {
+    const employeeUpsert = await prisma.employee.upsert({
+      where: { email: employee.email },
+      update: {
+        role: Role.Admin,
+        firstName: employee.firstName,
+        lastName: employee.lastName,
+        emailVerified: new Date().toISOString(),
```
Comment (OustanDing) [inline prisma/seed.ts:64]: Great catch 😄

**Review by OustanDing — APPROVED** [review]
LGTM!

File: prisma/seed.ts (hunk 3 above)
Comment (j-bunk) [reply prisma/seed.ts]: Done! Also did some minor refactoring to not repeat `devSeed` code

**Pull request merged**